src/lib/updater/index.js

```diff
@@ -0,0 +1,120 @@
+import * as logger from '../logger';
+import * as installer from '../installer';
+import * as spawnPromise from '../spawn-promise';
+var util = require('util')
+  , fs = require('fs')
+  , SpawnPromise = spawnPromise.SpawnPromise;
+
+function update(options) {
+  if (options.nuclear){
+    logger.log('Going NUCLEAR!')
+    logger.log('Clearing jspm packages...');
+    moveJSPMPackages(function(){
+      logger.log('Clearing npm packages...');
+      moveNPMPackages(function(){
+        normalUpdate();
+      });
+    });
+  } else {
+    logger.log('Updating normally...');
+    normalUpdate();
+  }
+}
+
+function unbundle(){
+  logger.log('Running jspm unbundle');
+  return SpawnPromise({
+    command: 'jspm',
+    args: ['unbundle']
+  });
+}
+
+function normalUpdate(){
+  var updateCommands = [];
+  var repoList = readConfigJs();
+  unbundle().then(function(){
+    installer.runNPMInstall(function(){
+      logger.log('Successfully npm installed');
+      logger.log('Updating all aurelia libs');
+      repoList.forEach(repo => {
+        updateCommands.push({ command: 'jspm', args: ['install', repo]})
+      });
+      SpawnPromise(updateCommands)
```

Comment:
Guess this is what you meant with the synchronous multi execution. Guess we should, as mentioned in the gitter channel, use something like Bluebirds Promise.join feature